doc/source/virtual-cluster/key-concepts.rst

```diff
@@ -0,0 +1,62 @@
+Key Concepts
+============
+
+.. _virtual-cluster-key-concepts:
+
+This page introduces key concepts for Ray virtual clusters:
+
+.. contents::
+    :local:
+
+Virtual Cluster
+---------------
+**Virtual cluster** provide logical resource isolation, enable multi-tenancy by allowing
+Ray cluster admins to easily manage multiple workloads on the same Ray cluster.
+
+A Virtual Cluster consists of a group of heterogeneous worker nodes or a group of
+child clusters :ref:`child cluster <virtual-cluster-divisible-cluster>`
+and :ref:`indivisible cluster <virtual-cluster-indivisible-cluster>`:
+
+.. image:: images/ray-virtual-cluster.png
+    :align: center
+    :width: 800px
+
+Users can submit jobs for execution on the Ray virtual cluster, or can interactively use the
+virtual cluster by connecting to the head node and running `ray.init`. See
+:ref:`Ray Jobs <jobs-quickstart>` for more information.
```

Comment:
Users can submit jobs for execution on the Ray virtual cluster through the HTTP API or by using command-line tools.